lib/matplotlib/pyplot.py

```diff
@@ -426,43 +426,43 @@ def figure(num=None,  # autoincrement if None, else integer from 1-N
         `num`.
 
     figsize : tuple of integers, optional, default: None
-        width, height in inches. If not provided, defaults to rc
-        figure.figsize.
+        width, height in inches. If not provided, defaults to 
+        :rc:`figure.figsize`.
```

Comment:
It was suggested in another thread, that we add the default rcParams value in brackets, such as
~~~
defaults to :rc:`figure.figsize` (``[6.4, 4.8]``)
~~~

I think that's a reasonable addition. While `:rc:` could be extended to write that as well, there's a benefit in having the value within the docstring itself.

Reply:
This was just suggested in another thread. We do not have a convention yet and I have no clear idea how this formatting should be. Maybe ``rcParams["figure.figsize"] = [6.4, 4.8]``?. Do you have any suggestions?